ui/chat/src/discussion.ts

```diff
@@ -106,6 +113,13 @@ const setupHooks = (ctrl: Ctrl, chatEl: HTMLElement) => {
     })
   );
 
+  if(sessionStorage.getItem("txt") != null && sessionStorage.getItem("txt") != "") {
+    chatEl.focus();
+    var event = new KeyboardEvent("keydown", {key : " "});
+    chatEl.dispatchEvent(event);
```

Comment:
i don't think you should simulate an event to reset the chat, best to reset the chat directly